packages/vite/src/node/server/index.ts

```diff
@@ -615,6 +615,8 @@ async function startServer(
       )
 
       printServerUrls(hostname, protocol, port, base, info)
+      // print shortcuts info
+      info(`  > Shortcuts: "r" restart, "o" open browser, "f" force restart`)
```

Comment:
@antfu "force restart" seems too vague? Should we explain it in the docs?

Reply:
Yeah, think we should document it, as we could possibly have more shortcuts in the future but not necessary to be print out here.